assets/scss/components/_facet.scss

```diff
@@ -0,0 +1,58 @@
+.c-facets {
+    width: 100%;
+}
```

Comment:
@mgbelegu this is a perfect candidate for a utility class like `u-width-full` or something similar.